openmdao/core/problem.py

```diff
@@ -171,6 +171,12 @@ class Problem(object):
         publishing work that uses this class.
     options : <OptionsDictionary>
         Dictionary with general options for the problem.
+    model_options : dict
+        A  dictionary of options to be passed to subsystems in the problem's model.
+        This dictionary is keyed by a path pattern string, and the associated value for each path
+        pattern is a dictionary of {option_name: option_val}. Those subsystems within the
+        hierarchy which match the path pattern and that have an option of the given name, will
+        have the value of that option overridden by value given in the dictionary.
```

Comment:
should probably clarify that this happens at setup time..  setting an option after setup will have no effect...